src/notation/internal/notationproject.cpp

```diff
@@ -36,6 +42,7 @@ NotationProject::NotationProject()
 {
     m_engravingProject = EngravingProject::create();
     m_masterNotation = std::shared_ptr<MasterNotation>(new MasterNotation());
+    m_projectAudioSettings = std::shared_ptr<ProjectAudioSettings>(new ProjectAudioSettings());
```

Comment:
would be slightly better to use std::make_shared

Reply:
not this case, I did `ProjectAudioSettings` constructor private (for controlling who can create it), so we can't use `std::make_shared` in such cases